src/components/application_manager/src/commands/mobile/add_command_request.cc

```diff
@@ -318,17 +320,24 @@ void AddCommandRequest::on_event(const event_engine::Event& event) {
       if (hmi_apis::Common_Result::SUCCESS != ui_result_) {
         (*message_)[strings::msg_params].erase(strings::menu_params);
       }
+      if (message[strings::msg_params].keyExists(strings::info)) {
+        info_ += message[strings::msg_params].asString();
```

Comment:
@LuxoftAKutsan isn't correct string please change to "message[strings::msg_params][strings::info].asString()